helm-chart/test.values.yaml

```diff
@@ -12,26 +12,62 @@ cvat:
         - name: DJANGO_SETTINGS_MODULE
           value: cvat.settings.testing_rest
     worker:
-      import:
-        replicas: 1
-        additionalVolumeMounts:
-        - mountPath: /home/django/share
-          name: cvat-backend-data
-          subPath: share
-      export:
-        replicas: 1
+      analytics_reports:
+        replicas: 0
+      annotation:
+        replicas: 0
       chunks:
+        replicas: 0
+      consensus:
+        replicas: 0
+      export:
+        replicas: 0
+      import:
+        replicas: 0
+      webhooks:
+        replicas: 0
+      utils:
         replicas: 1
+        additionalEnv:
+          - name: DJANGO_SETTINGS_MODULE
+            value: cvat.settings.testing_rest
+          - name: CLICKHOUSE_HOST
+            valueFrom:
+              secretKeyRef:
+                name: cvat-analytics-secret
+                key: CLICKHOUSE_HOST
+          - name: CLICKHOUSE_PORT
+            valueFrom:
+              secretKeyRef:
+                name: cvat-analytics-secret
+                key: CLICKHOUSE_PORT
+          - name: CLICKHOUSE_USER
+            valueFrom:
+              secretKeyRef:
+                name: cvat-analytics-secret
+                key: CLICKHOUSE_USER
+          - name: CLICKHOUSE_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: cvat-analytics-secret
+                key: CLICKHOUSE_PASSWORD
```

Comment:
Can we just set these variables for all workers? I don't see a reason to micromanage them.